pyqg/xarray_output.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import xarray as xr
+
+def model_to_dataset(m):
+    '''Convert outputs from model to an xarray dataset'''
+
+   # Define dict for variable dimensions
+    spatial_dims = ('time','z','y','x')
+    spectral_dims = ('time','z','l','k')
+    dim_database = {
+        'q': spatial_dims,
+        'u': spatial_dims,
+        'v': spatial_dims,
+        'ufull': spatial_dims,
+        'vfull': spatial_dims, 
+        'qh': spectral_dims,
+        'uh': spectral_dims,
+        'vh': spectral_dims,
+        'ph': spectral_dims, 
+        'Ubg': ('z'),
+        'Qy': ('z'),
+    }
+
+    # dict for variable dimensions
+    var_attr_database = {
+        'q': {'long_name': 'potential vorticity in real space', 'units': 'm\u00B2 s\u207B\u00B9 K kg\u207B\u00B9',},
```

Comment:
What are these symbols? Is this some kind of encoding?

Let's use [CF UDUNITS conventions)[https://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch03.html} for the units.